jujugui/static/gui/src/app/components/added-services-list/added-services-list.js

```diff
@@ -24,7 +24,7 @@ YUI.add('added-services-list', function() {
 
     generateItemList: function(services) {
       var items = [];
-      services.forEach(function(service) {
+      services.forEach((service) => {
```

Comment:
Not at chance :) 

Babel converts it to a bound function in the output - yay for a build process!